a/animal-sniffer/animal-sniffer_ubuntu_20.04.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : animal-sniffer
+# Version       : master
+# Source repo   : https://github.com/mojohaus/animal-sniffer
+# Tested on     : Ubuntu 20.04.1 LTS (Focal Fossa)
+# Script License: MIT License
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+
+#Install Updates
+sudo apt-get clean
+sudo apt-get update
+sudo apt-get install maven
+
+#Build and test animal-sniffer package
+if [ $# -lt 1 ]
+then
+  echo "Usage: $0 branch or tag_name"
```

Comment:
Will it accept both branch name and tag name ? Also, it should build a default version if we don't pass any argument, as per the requirement. Pl. also upload the license file.